products/extensions/b2b-components/shopping-lists/01-entities-and-schema.md

```diff
@@ -0,0 +1,41 @@
+---
+nav:
+  title: Entities & Schema
+  position: 20
+
+---
+
+# Entities and schema
```

Comment:
@leduc92 , @nguyenky - I would like to discuss about the document structuring w.r.t B2B components.

Reply:
After conversation with Duc and Nam, yesterday. I will rearrange the structure as per this : https://shopware.atlassian.net/browse/DX-592. After that will restructure this PR too.